src/ray/gcs/gcs_client/service_based_gcs_client.cc

```diff
@@ -146,7 +146,7 @@ void ServiceBasedGcsClient::PeriodicallyCheckGcsServerAddress() {
       RayConfig::instance().gcs_service_address_check_interval_milliseconds());
   detect_timer_->expires_from_now(check_period);
   detect_timer_->async_wait([this](const boost::system::error_code &error) {
-    if (error == boost::system::errc::operation_canceled) {
+    if (error == boost::asio::error::operation_aborted) {
       // `operation_canceled` is set when `detect_timer_` is canceled or destroyed.
```

Comment:
Comments are also modified at the same time?

Reply:
done